website/docs/language/values/variables.mdx

```diff
@@ -234,7 +234,7 @@ disclosing the content of one block might imply the content of a sibling block.
 ```
 
 A provider can also
-[declare an attribute as sensitive](/docs/plugin/sdkv2/best-practices/sensitive-state#using-the-sensitive-flag),
+declare an attribute as sensitive,
```

Comment:
maybe we should link to the provider sdk docs here? what do others think?

Reply:
@opentofu/core-engineers ?

Reply:
So we decided we'll link to the provider SDK docs here